styles/button.css

```diff
@@ -61,3 +61,7 @@
   outline: 2px solid var(--bs-btn-focus-box-shadow); /* Outer color */
   box-shadow: 0 0 0 4px var(--stanford-black); /* Inner color */
 }
+
+.btn-group .dropdown-menu {
```

Comment:
I don't think we need the `.btn-group` scoping either. I think we would like this for all dropdowns.

Reply:
My assumption was we needed it. If there isn't a nearby button then --bs-btn-color isn't set. I could see this happening with a link that has a dropdown.